dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ESContentletAPIImpl.java

```diff
@@ -5180,19 +5180,16 @@ else if(field.getFieldType().equals(Field.FieldType.DATE_TIME.toString())){
 
                     buffy.append(" +(live:true working:true)");
```

Comment:
I don't know if we handle searching both live and working in the same query - we might have to do two queries to be sure.

Reply:
done